src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -372,28 +409,54 @@ public void validateMedicalInventoryRow(MedicalInventory inventory, List<Medical
 	 *
 	 * @param inventory the {@link MedicalInventory}
 	 * @param inventoryRowSearchList the list of {@link MedicalInventory}
+	 * @param allMedicals if {@code true} will check also medicals not in {@code inventoryRowSearchList}
 	 * @throws OHServiceException
 	 */
-	public void validateMedicalWardInventoryRow(MedicalInventory inventory, List<MedicalInventoryRow> inventoryRowSearchList) throws OHServiceException {
+	public void validateMedicalWardInventoryRow(MedicalInventory inventory, List<MedicalInventoryRow> inventoryRowSearchList, boolean allMedicals)
+		throws OHServiceException {
 		LocalDateTime movFrom = inventory.getInventoryDate();
 		LocalDateTime movTo = TimeTools.getNow();
-		StringBuilder medDescriptionForLotUpdated = new StringBuilder("\n"); // initial new line
-		StringBuilder medDescriptionForNewLot = new StringBuilder("\n"); // initial new line
-		StringBuilder medDescriptionForNewMedical = new StringBuilder("\n"); // initial new line
+		StringBuilder medDescriptionForLotUpdated = new StringBuilder('\n'); // initial new line
+		StringBuilder medDescriptionForNewLot = new StringBuilder('\n'); // initial new line
+		StringBuilder medDescriptionForNewMedical = new StringBuilder('\n'); // initial new line
 		boolean lotUpdated = false;
 		boolean lotAdded = false;
 		boolean medicalAdded = false;
+		boolean dateUpdated = false;
```

Comment:
```suggestion
		boolean dateUpdated;
```
The variable always has a value set so setting to `false` is redundant.

Reply:
ok, thanks!